tests/test_transpile.py

```diff
@@ -3,6 +3,7 @@
 from unittest import mock
 
 from sqlglot import parse_one, transpile
+from sqlglot.dialects import ClickHouse, Oracle
```

Comment:
No need to import dialects, just use the string version, i.e. `"clickhouse"`, and `"oracle"`.